packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -130,8 +131,23 @@ export class MediaPlaceholder extends Component {
 				setMedia = onSelect;
 			}
 		} else {
+			const [ file ] = files;
+
+			if ( file.type === 'image/svg+xml' ) {
+				file.text().then( ( source ) => {
+					this.props.onSelectURL(
```

Comment:
Should we check if onSelectURL is available?

Reply:
Oh, sure, I didn't know it's optional. :)